nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/docs/configuration/local.md

```diff
@@ -25,18 +25,18 @@ The public docker images are tagged with the same version numbers as the code, w
 ## Singularity image
 Many HPC environments are not able to run Docker due to security issues. [Singularity](http://singularity.lbl.gov/) is a tool designed to run on such HPC systems which is very similar to Docker. Even better, it can use create images directly from dockerhub.
 
-To use the singularity image for a single run, use `-with-singularity 'docker://{{ cookiecutter.pipeline_slug }}'`. This will download the docker container from dockerhub and create a singularity image for you dynamically.
+To use the singularity image for a single run, use `-with-singularity 'docker://{{ cookiecutter.dockerhub_slug }}'`. This will download the docker container from dockerhub and create a singularity image for you dynamically.
```

Comment:
shub:// instead - also make sure the dockerhub slug is even correct. On singularity hub, we are using shub://nf-core/<repo> instead of docker://nfcore/<repo>

Reply:
As above, should be `-profile singularity` instead of `-with-singularity`